rcl/src/rcl/node_options.c

```diff
@@ -32,6 +32,7 @@ rcl_node_get_default_options()
     .use_global_arguments = true,
     .enable_rosout = true,
   };
+  default_options.rosout_qos = rmw_rosout_qos_profile_default;
```

Comment:
this can be static member? i think that it does not need to store every single time but statically initialized as default_options member along with enable_rosout.

Reply:
Static member `default_options` need to be initialized with constants, `rmw_rosout_qos_profile_default` cannot be used to initialize it in c language.

Reply:
When build rcl in my develop environment, `Ubuntu 18.04, gcc 7.5`, build error as below
```
/home/dbt/tmp/workspace/src/rcl/rcl/src/rcl/node_options.c:34:19: error: initializer element is not constant
     .rosout_qos = rmw_rosout_qos_profile_default,
```
While today i found it can build successfully in Ubuntu20.04 which has a gcc of more advanced version, So for backward compatibility, it's better to use assignment method.

Reply:
you can delete the `static` above, and make proper use of designated initializers.
There's no benefit on using `static` in the above definition.

Reply:
Ok, i will change it.